docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,204 @@
+
+* Product
+* Category
+* Brand
+* NormalPage
+* ContactPage
+* RawHtmlPage
+* Blog
+* BlogPost
+
+## Enhanced redirect support
+
+The new `LookUpUrl` capability now supports redirects, helping you determine the appropriate page to display. If a redirect exists for a given path, the `Redirect` node within the `route` query returns the redirect details. Otherwise, it returns `null`.
+
+## Redirect enum values
+
+The `redirectBehavior` argument supports two values:
+
+* **IGNORE**(default): 
+
+   * The Storefront GraphQL API disregards redirects and prioritizes custom URLs. 
+   * If both a redirect and an entity URL exist for the same path, both the `redirect` and `node` nodes return non-null values. 
+   * This behavior ensures backward-compatiblity.
+
+* **FOLLOW**: 
+
+  * For dynamic or association redirects, the `node` node returns the target entity (e.g., category, product, blog, etc.). 
+  * For static/manual redirects, the `node` node returns `null` (since there’s no associated entity), while the `redirect` node provides the redirect details.
+
+## Example queries
+
+### Query: Fetching route details
+
+This example retrieves details about an object (e.g., Product, Brand, or Category) based on a given URL path.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Query route details" showLineNumbers copy
+
+     query LookUpUrl(
+       $urlPath: String!
+       # Use GraphQL Query Variables to provide a path
+     ) {
+       site {
+         route(path: $urlPath) {
+           node {
+             __typename
+             id
+             # A different response is returned based on which type of object was matched
+             ... on Category {
+               name
+               description
+             }
+             ... on Brand {
+               name
+               defaultImage {
+                 url(width: 200)
+               }
+             }
+             ... on Product {
+               name
+               description
+               images {
+                 edges {
+                   node {
+                     url(width: 500, height: 500)
+                   }
+                 }
+               }
+             }
+             ... on Redirect {
+               # all redirect attributes
+               toUrl
+             }
+           }
+         }
+       }
+     }
+```
+```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "urlPath": "/blog/"
+    }
+```
+  </Tab>
+  <Tab>
+    ```json filename="Example response" showLineNumbers copy
+     {
+        "data": {
+          "site": {
+            "route": {
+              "node": {
+                "__typename": "Blog",
+                "id": "QmxvZzoz"
+              }
+            }
+          }
+        }
+      }
+   ```
+</Tab>
+</Tabs>
+
+### Query: Using `IGNORE` redirect behavior
+
+In this example query, the `node` value is "null" because the redirect is ignored.
```

Comment:
I think it would be helpful to add a note here, ie:

> In this example query, the `node` value is "null" because the redirect is ignored. Note that the `redirectBehavior: IGNORE` input is default so could be excluded from the query and the same results would be received.